test/subscriptions.spec.ts

```diff
@@ -133,25 +133,25 @@ describe('subscriptions-high-level-flows', async () => {
     expect(await d.erc20.balanceOf(subscriber.address)).is.bignumber.equal('0');
     expect(await d.erc20.balanceOf(d.subscriptions.address)).is.bignumber.equal('0');
 
-    expect(await d.erc20.balanceOf(d.rewards.address)).is.bignumber.equal(firstPayment.add(secondPayment));
+    expect(await d.erc20.balanceOf(d.certifiedFeesWallet.address)).is.bignumber.equal(firstPayment.add(secondPayment));
   });
 
   it('adds and removes subscriber only by functional owner', async () => {
     const d = await Driver.new();
     const subscriber = await d.newSubscriber('tier', 1);
 
-    await expectRejected(d.subscriptions.addSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), "Non-owner should not be able to add a subscriber");
+    await expectRejected(d.subscriptions.addSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), /caller is not the functionalOwner/);
     let r = await d.subscriptions.addSubscriber(subscriber.address, {from: d.functionalOwner.address});
     expect(r).to.have.a.subscriberAddedEvent({subscriber: subscriber.address})
 
-    await expectRejected(d.subscriptions.removeSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), "Non-owner should not be able to remove a subscriber");
+    await expectRejected(d.subscriptions.removeSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), /caller is not the functionalOwner/);
     r = await d.subscriptions.removeSubscriber(subscriber.address, {from: d.functionalOwner.address});
     expect(r).to.have.a.subscriberRemovedEvent({subscriber: subscriber.address})
   });
 
   it('should not add a subscriber with a zero address', async () => {
     const d = await Driver.new();
-    await expectRejected(d.subscriptions.addSubscriber(ZERO_ADDR, {from: d.functionalOwner.address}), "Should not deploy a subscriber with a zero address");
+    await expectRejected(d.subscriptions.addSubscriber(ZERO_ADDR, {from: d.functionalOwner.address}), /must provide a valid address/);
```

Comment:
unnecessary check, in particular the check in the removeSubscriber

Reply:
Right, removing